docs/00-getting-started.md

```diff
@@ -53,15 +59,17 @@ Once you have one, simply log in and go to the [Azure Portal](https://portal.azu
 
 4. Save your account credentials into the secrets.yaml file
 
-- Copy the secrets.yaml.template file to secrests.cfg
+- Copy the secrets.yaml.template file in the .thunderbolt/ directory to .thunderbolt/secrets.yaml
 
 Windows
 ```sh
+cd .thunderbolt
```

Comment:
I think the azb spark init should also rename that file for you and later(other pr) we can have it ask for the fields as they run `azb init`

Reply:
good point, it should definitely rename the file.